client/src/pages/PackageDetailsScreen/index.ts

```diff
@@ -0,0 +1,3 @@
+import PackageDetailsScreen from './PackageDetailsScreen';
+
+export { PackageDetailsScreen };
```

Comment:
```suggestion
export * from './PackageDetailsScreen';
```
just do that. it is cleaner.